docs/documentation/index.html

```diff
@@ -2700,6 +2700,12 @@ <h3 id="core-cache">Core cache</h3>
           negative on CI checks <br>
           <span class="tag">Default:</span> <code>120000</code>
         </dd>
+
+        <dt><code>MO_INITIALMAXDELAY</code> <code class="type">number</code></dt>
+        <dd>
+          Duration in milliseconds is used to add delay on first method to avoid bursting x requests in same time on start <br>
```

Comment:
```suggestion
          Duration in milliseconds used as the maximum delay on each tile's first update to avoid bursting N requests at the same time on start <br>
```